### DISCUSSION TITLE
blog/2d85b9f8-a6c9-4252-a8f9-27e18069026a

### DISCUSSION THREAD

**Discussion by giscus** [CaesiumY/notion-blog-v2#7] [giscus]
# blog/2d85b9f8-a6c9-4252-a8f9-27e18069026a

http://localhost:3000/blog/2d85b9f8-a6c9-4252-a8f9-27e18069026a

**Comment by CaesiumY** [comment]
hello world

**Comment by MyNameSieun** [comment]
### 블로그가 너무 깔끔하고 예쁘네요!!! (❁´◡`❁)

**Reply by CaesiumY** [reply]
하하핳 강의용으로 만든 거라 미숙한 플젝입니다😊
